pkg/ebpf/probes/cgroup.go

```diff
@@ -0,0 +1,87 @@
+package probes
+
+import (
+	"fmt"
+	"os"
+
+	bpf "github.com/aquasecurity/libbpfgo"
+)
+
+//
+// Cgroup
+//
+
+type cgroupProbe struct {
+	programName string
+	attachType  bpf.BPFAttachType
+	bpfLink     *bpf.BPFLink
+}
+
+// attach attaches an eBPF program to a cgroup
+func (p *cgroupProbe) attach(module *bpf.Module, args ...interface{}) error {
```

Comment:
Yes, its explained in pkg/ebpf/probes/probes.go:

```
//
// Probes:
//
// NOTE: keeping both (Probes and Probe) interfaces with variadic args on
//       **purpose** until we define real use cases, by extending supported
//       "probes" types (trace, tc, socket, xdp, tunnel, cgroup, ...) **
//
```

All attaches are the same and keep a variadic arg because some implementations need network interfaces, some others need cgroupfs, some don't need anything.

I'll add that comment to all files from the probes directory, so it is clear why the args is like that.